pkg/konnector/controllers/cluster/serviceexport/cluster-scoped/utils.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2023 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterscoped
+
+import (
+	"errors"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// ClusterNsAnnotationKey is the annotation key to identify the cluster namespace that corresponds to
+// a cluster-scoped object. Its value is the corresponding cluster namespace name.
+const ClusterNsAnnotationKey = "kube-bind.io/cluster-namespace"
```

Comment:
be clear where this is used. On the client-side objects?

Reply:
Clarified this, also squashed into the commit that implements the shim.